src/dbnode/persist/fs/files.go

```diff
@@ -292,6 +298,55 @@ func timeAndIndexFromFileName(fname string, componentPosition int) (time.Time, i
 	return t, int(index), nil
 }
 
+// SnapshotTime returns the time at which the snapshot was taken.
+func SnapshotTime(
```

Comment:
See if I can just include this in the return from SnapshotFiles()

Reply:
+1

Reply:
done